bin/test

```diff
@@ -6,5 +6,8 @@ if [ $(sysctl -n hw.logicalcpu) != "1" ] && ! psql "postgresql://localhost/vita-
   exit 1
 fi
 
+# Increase the # of max open files to avoid https://stackoverflow.com/questions/59890432/rack-error-runtimeerror-failed-to-get-urandom-in-a-rails-app-rails-5-0-6-ru
+ulimit -n 524288
```

Comment:
if i were being really nitpicky i'd say to put this in a separate commit

Reply:
Done! :) Removed that from this PR.